iroha_2_whitepaper.md

```diff
@@ -70,8 +70,14 @@ Iroha provides a library of smart contracts called **I**roha **S**pecial **I**ns
 These include the following smart contracts to support asset management use cases:
 
 * Domain registration
-* Asset creation
-* Asset Transfer
+* createAsset
+* transferAsset
+* addAssetQty
+* subAssetQty
+* groupAssets
+* ungroupAssets
+* swapAssets
+* invoiceAsset
```

Comment:
It will be more descriptive to provide not a names of contracts, but definitions:
```
* Create new type of an Asset
* Transfer some amount of defined Asset from one Account to another
* Add defined amount to the quantity of defined Asset in Account (BTW - do we really need different commands to create Asset and to add Quantity?)
...
*